QUEST_LV_0300.txt

```diff
@@ -17,47 +17,47 @@ QUEST_LV_0300_20150717_000016
-QUEST_LV_0300_20150717_000041
-QUEST_LV_0300_20150717_000042
-QUEST_LV_0300_20150717_000043
-QUEST_LV_0300_20150717_000044
-QUEST_LV_0300_20150717_000045
-QUEST_LV_0300_20150717_000046
-QUEST_LV_0300_20150717_000047
-QUEST_LV_0300_20150717_000048
-QUEST_LV_0300_20150717_000049
-QUEST_LV_0300_20150717_000050
-QUEST_LV_0300_20150717_000051
-QUEST_LV_0300_20150717_000052
-QUEST_LV_0300_20150717_000053
-QUEST_LV_0300_20150717_000054
-QUEST_LV_0300_20150717_000055
-QUEST_LV_0300_20150717_000056
-QUEST_LV_0300_20150717_000057
-QUEST_LV_0300_20150717_000058
-QUEST_LV_0300_20150717_000059
-QUEST_LV_0300_20150717_000060
+QUEST_LV_0300_20150717_000020 $Repairing/MAKING/3/TRACK_AFTER/None
+QUEST_LV_0300_20150717_000021 $Check detector(5)
+QUEST_LV_0300_20150717_000022 $Repairing/MAKING/3/TRACK_BEFORE/None
+QUEST_LV_0300_20150717_000023 $Notice/!/Parts are not fixed{nl}You need rope to tighten up!#5
+QUEST_LV_0300_20150717_000024 $Notice/!/You need rope to tighten up#5
+QUEST_LV_0300_20150717_000025 $For Guiding Souls
+QUEST_LV_0300_20150717_000026 $It's okay to be relieved now
+QUEST_LV_0300_20150717_000027 $Checking the device/LOOK/2/TRACK_BEFORE/None
+QUEST_LV_0300_20150717_000028 $Notice/!/As you touch the device, you feel like your soul is escaping!{nl}You are gradually losing energy!
+QUEST_LV_0300_20150717_000029 $Notice/You are recovered now#5
+QUEST_LV_0300_20150717_000030 $All I hope is only...(1)
+QUEST_LV_0300_20150717_000031 $About the reason of hoping for death
+QUEST_LV_0300_20150717_000032 $All I hope is only...(3)
+QUEST_LV_0300_20150717_000033 $All I hope is only...(2)
+QUEST_LV_0300_20150717_000034 $Tell him that you are going to meet Cleric Master
+QUEST_LV_0300_20150717_000035 $You can't cooperate anymore
+QUEST_LV_0300_20150717_000036 $Destroy Terminal(1)
+QUEST_LV_0300_20150717_000037 $About Main Body and Terminal
+QUEST_LV_0300_20150717_000038 $Notice/!/Avoid monsters' attacks and destroy Terminal!#5
+QUEST_LV_0300_20150717_000039 $Destroy Terminal(2)
+QUEST_LV_0300_20150717_000040 $Igniting explosive powder/FIRE/2/TRACK_BEFORE/None
+QUEST_LV_0300_20150717_000041 $Notice/Terminal is destroyed by the explosion!#5
+QUEST_LV_0300_20150717_000042 $Destroy Terminal(3)
+QUEST_LV_0300_20150717_000043 $Eternal Repose(1)
+QUEST_LV_0300_20150717_000044 $Last thing to say
+QUEST_LV_0300_20150717_000045 $Eternal Repose(2)
+QUEST_LV_0300_20150717_000046 $Placing Orb/MAKING/3/TRACK_BEFORE/None
+QUEST_LV_0300_20150717_000047 $Eternal Repose(3)
+QUEST_LV_0300_20150717_000048 $Say bye
+QUEST_LV_0300_20150717_000049 $Notice/Svitrigaila's hope is granted#5
+QUEST_LV_0300_20150717_000050 $Check detector
+QUEST_LV_0300_20150717_000051 $Talk to Kupole 메일레
+QUEST_LV_0300_20150717_000052 $Talk to Kupole who is rescued from Petrification Frost.
+QUEST_LV_0300_20150717_000053 $Check Petrification detectors
+QUEST_LV_0300_20150717_000054 $To help Kupole 메일레 to guide souls safely, check detectors that you have not checked in this area.
```

Comment:
"Check the detectors in this area to help Kupole Meile guide the souls safely from the Stone Frost."